src/book/03-guides/04-general/04-code-conventions.mdx

```diff
@@ -1,12 +1,26 @@
 ---
 section: Guides
 chapter: General
-title: Code Documentation
-description: How to use documentation tags
-slug: /guides/general/documentation
+title: Code Conventions
+description: Conventions used in NUbots code.
+slug: /guides/general/code-conventions
 ---
 
-## Conventions
+## Naming Conventions
```

Comment:
If this is a section called naming conventions, please link to the part where we explain our convention for naming homogeneous matrices

Reply:
@KipHamiltons let me know if the maths part is good.